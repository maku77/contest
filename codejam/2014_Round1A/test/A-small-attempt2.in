100
1 2
00
10
10 10
1100010011 0101110100 1011100001 0110111111 0010000001 1011101100 1111001110 1000001001 0001110010 0100011110
1001011000 1010010011 1101100110 0100001011 0000101001 1011111001 0011110100 0100000110 1110010101 0111101110
8 10
0110001000 0110011010 1001100001 0110001010 1001110011 1001100011 1001110111 0110011110
1001110001 0110011100 0110011000 0110001100 1001100101 1001110101 1001100111 0110001110
10 10
0011100111 1100101001 1000011011 1001011100 1111100010 0000100000 1110000100 0010010101 0111011011 0101111110
0100111100 0110011001 1111000110 1000011110 1110100000 1001011001 1101101011 0011010111 0010100101 0001100010
4 6
110100 010100 011011 111011
110100 111011 011011 010100
1 3
001
101
10 10
1110011110 1101100100 0010111010 0000110101 0100000011 0111001101 1001110001 1111001000 1011111011 0000000110
0111010101 1001111110 1010000100 1110010001 1000101000 0101011010 0011100011 0111100110 1100011011 0000101101
1 2
11
11
2 9
100010001 111001111
100110110 011001010
10 10
0011110101 1110011111 1100011111 1101000110 1111010011 1010010000 0000101000 0100101001 0001101010 0011100100
1010110001 1100111110 0101001010 1000110001 1011101000 1001111101 0101011011 0110000110 0010000111 0111000100
6 8
00001101 01110000 10110011 00101100 01101101 10010011
00010101 10101011 11010110 00110101 10001010 11001011
10 10
1000001101 0001011100 0011110100 1110100110 1000101111 1001011010 0110010010 1111100011 0100111001 0111000001
0011110111 0001011111 1000001110 0110010001 0111000010 1001011001 1000101100 1110100101 1111100000 0100111010
1 7
0001011
1100111
10 10
0110100000 0100100011 0011000101 1001111110 1001010111 1111101111 0010010010 1100011000 0101101101 1010011000
0000010111 0101110001 0101011000 0011100000 0110010111 1010101111 1111001010 1000101100 1001100010 1110011101
9 7
0101100 0111011 1001000 0110001 0001011 0011011 1111111 0111111 1100100
1011101 1110010 0100001 1111110 0101101 1001001 1011010 0000010 1111011
5 9
000111001 110100100 101100110 000000001 011101011
101000111 001011010 101010000 001011101 100010100
5 7
1110000 1001001 1100011 1110011 1110100
1000001 1101011 1111100 1111000 1111011
5 9
110110100 001101001 111100111 111011100 111001100
010100011 011011011 010110011 010001000 100000110
10 10
0001110110 0000111110 1101011011 1011001001 1010101110 1010010100 0110101000 1110110011 0101000001 0101000101
0100001100 1111101110 0011000011 1011011001 1011011101 1000110000 1110100110 0100110110 0101010001 0000101011
10 10
1000010000 1011010011 0001101010 0001011101 1110001111 0100111100 1111100001 0001101000 0110110111 1110000110
1000011000 0001100010 1111101001 1110001110 1110000111 0110111111 0100110100 1011011011 0001010101 0001100000
4 4
1101 1100 0100 0111
0100 1101 1100 0101
2 5
10001 10011
10010 10111
6 8
01010011 00111011 00100100 00101101 10100111 10010010
10101110 00000000 01000110 01110000 00011000 00100011
4 4
0000 1111 0110 1001
0000 0110 1111 1001
3 9
100111100 011100101 100011010
111011100 010011101 110011101
1 2
01
11
4 6
111011 001000 101011 101110
001110 000100 011111 010011
10 8
11000000 11001110 10111111 11000101 11100101 11110100 11011011 00110010 10000000 00010101
10100010 10100111 11100010 10000111 10010110 01110111 11011101 10111001 01010000 10101100
4 9
011111000 110100111 110101001 010110011
101001001 100011100 101101010 001101100
10 10
0000010001 1001100110 0011111111 0100011100 1100100010 0010001100 1110010011 0101101011 1111000100 1011111001
1001000110 1011011011 0100001110 0111101000 0010101100 0101011001 1000110101 1110100001 0000110011 1111010110
2 3
111 001
001 011
10 10
1010000100 0000110010 0111011001 0010000100 1100000101 1111110111 0100101011 1001111000 1001011010 0111101111
0000100000 1011001010 1000111000 0111111101 0011100100 0000010110 0101001011 1110110111 1110010101 1101001011
10 10
0001010101 1011110010 0010010010 1110011000 0111111111 1001100101 1100001011 1010101100 0101001000 0100100111
0011010000 1011100000 1111111101 0000011001 0100101101 0110111110 1000100111 1110010010 1101001010 0001000111
8 9
001100100 010001000 110100011 111000111 010100000 011000100 101100111 110001011
011001000 110101111 111001011 100001111 011100000 010101100 111100011 000001100
3 9
101001110 100000100 000101001
010110011 100100100 111110001
10 10
1000011001 0101101010 0001010110 0110001000 0000100100 1011001001 1111111111 1010100110 0100010011 1111110101
0111111101 0011001010 1001001000 1100010001 0010111000 1011110111 0101100110 1100011011 0110000100 1000100111
10 10
1010110011 0011011001 0100110110 0010100100 1110001110 0001001000 0101011101 1101000111 1010100010 1101111001
1000101100 1001010001 0111001111 0000101010 1111010101 0000111011 0111110001 0100000110 1110111110 1011000000
6 4
0100 1010 1100 1000 0011 0000
1011 0111 1101 0100 1111 0011
2 3
000 101
100 001
2 8
11010010 01011011
10000010 00001011
10 10
0010100111 1110110010 1111111010 1000010001 0101100011 0101101101 1001011100 0000001100 0010011000 1111000111
1011111011 0101000101 0000100100 0111101110 1010001110 0000101010 1101011000 1010110011 1100010101 0111010001
9 10
1000100100 1000100011 0000101000 0111010101 0100000001 1101111011 1100111101 0110100111 0011101111
1010011111 1110001110 1011000110 1011110001 1001110001 0010111000 0111000001 0111000100 1000100000
6 8
01010010 01111110 00101110 00101100 00000000 00000010
00010010 00010000 01000010 01101110 00111100 00111110
6 6
010101 001010 001011 110001 110100 000111
000001 100101 001011 111111 111010 011001
10 8
11100000 11111110 01101101 11001001 00110001 11101111 10011010 11001110 01100010 11000001
11000100 11001011 11000011 11100101 11110100 01101000 01100111 11101010 10010000 00111011
10 10
0111101010 1000001011 1111001111 1101011000 1000110111 0101111100 0100110111 1011110100 0010000001 0010000000
0110000101 0001111000 1000001101 1011001110 1110100001 1100110110 0111001110 1011110010 0100010011 0001111001
10 10
0110011001 1001000111 0011100101 0111000010 1000101110 1110001111 0100110011 1011111100 1100011000 0001110000
0100101000 0010011010 0001101011 0011000001 0110111101 1100011111 1110100100 1011010111 1001000000 1101110110
2 9
001011010 101011001
100001101 100001011
10 10
0100111011 1110001111 0001101100 0111110110 1000111101 1010000001 0111001010 1110010000 1001000100 0001110011
0100000010 1011110101 0010100100 1010001100 1001001001 0111110011 1101011000 0100111110 0010111011 1101000111
4 7
1111110 1111100 0110111 0110101
1011110 0010101 1011100 0010111
2 6
110001 100110
101110 111001
10 10
0111001110 0001110010 1111001001 0111010011 0010110100 1010100011 1101001101 0000110110 1100001001 1000111100
1001110110 0010100111 0100000110 1100011100 0001100001 1110011000 1011101001 0100011011 1111011100 0011100011
7 10
1100101000 1101000000 1110001000 0011100001 1001100101 0011111001 0110110010
1011110010 1100100001 0001001111 1000000000 0100011001 1110011011 1010010011
5 7
0011001 1101001 1000110 0011111 1011011
0111111 1001111 1111101 1100000 0111001
10 10
0001000100 1000110111 0010100010 1011011110 0110100000 1110101111 1011001000 1101111101 0100011001 0101010011
0110111010 1000100001 0101001001 0001001011 0010101101 1000110111 1011011110 1101000110 1110010100 0111110000
10 10
0100010111 1111001101 1010011011 0101110011 1001001000 1011101101 0110110000 0010101100 1001000010 0100110110
0000111111 1111000111 1101100010 1111001101 1100010100 0100100011 0010111001 0011111100 0010011000 1001000010
10 10
1000110010 1011101000 1010010111 0101001101 0111100001 0010001011 1111100011 0101110100 0000011100 1100011110
1110111111 0111101100 1010010011 1001001001 1000110110 1101000010 0000101010 0101000000 0111010101 0010111101
1 2
00
11
1 8
00000111
00000010
4 8
11110100 11001001 01000111 11111110
11110111 01001111 11000011 01001101
4 8
01000011 10101001 00101001 11000011
10010101 01111111 11111111 00010101
7 8
00011010 00010001 10101010 00001001 11001011 10100001 10001000
10110010 00111011 00001011 11001000 01001100 01110011 01010110
6 6
010000 110100 111001 011101 001011 001100
101011 001111 100110 110000 000010 110111
10 10
0000110000 1010101001 0101000110 0001101111 0110110111 1001011010 1110001111 1110110000 1011011000 0101000101
0010100111 0111010010 1001001111 0111010001 0100100000 1100100111 1011001101 1000111110 1100011000 0011111000
6 10
0101110111 1011001001 1011001000 1010001000 0100110110 0101110110
1010001000 1010001001 0100110111 0100110110 1011001000 0101110110
10 10
0000001001 1011011010 1110010000 0110101111 1011110101 1101110010 0101000100 0011000011 1100111100 0000101111
0100111001 1011000110 1101000001 0011011111 0101110111 1110101010 1000101010 1000001100 0011110000 0110010101
8 7
0100110 0011110 1011010 1001100 0010000 1111100 1000100 0000001
0001000 0101101 1111000 0010111 1110110 0100110 1011110 0110001
4 8
01101111 11010000 11001010 10101001
11010001 01101110 11001011 10101000
10 10
1100111110 1111011010 0011100111 0100110100 0100010010 1011100001 1000100001 0011001100 1100011111 0011001001
1011101011 0011000110 0100111000 1100010011 1011000000 1011101110 0100011001 0000000110 1100110101 0111111101
6 8
01101101 11000011 00111101 11010011 01111101 10010011
10010010 00111100 01111100 11000010 11010010 01101100
2 5
01110 11110
10011 00011
8 6
000010 101001 100100 111100 100000 001000 001011 111110
010010 100011 000100 110111 110011 101011 011111 010001
10 10
0010001101 1011000010 1011000001 0100001101 0010001110 0110111100 1111110011 1101000001 1111110000 0110111111
1011000001 1111110000 0110111100 1011000010 0010001101 1111110011 0010001110 0110111111 0000111100 1001110000
10 10
1100111111 0000000001 1101001001 0110000011 1001001000 0011111010 0010010110 1011110101 0110110110 1101101100
1011100011 1011010110 1111110110 0000101001 0000001100 0001011111 1101100001 1110011010 0110010101 0100101000
3 3
000 001 100
110 100 010
6 4
1111 0101 0001 0111 0011 1101
1001 0001 0011 1011 1111 1101
4 9
010111011 000010011 111110110 011111100
111101101 101000101 000001000 100000010
1 4
1001
0010
9 9
111011110 000110001 101101100 110011010 100111110 101111110 011001111 110111000 111010010
111001001 111111101 100010110 101110101 101100101 010101101 000010001 011010001 110100101
1 5
10010
11101
10 10
1010101100 1011010010 0110101111 0101100001 1011000101 0001000101 1110111010 0101110010 1000011111 0100011000
1001000011 1001010000 1010011110 0010001011 1101110100 1000101001 0110011101 0111110100 0111100011 0100101110
8 5
11110 00101 10100 10110 10001 11100 01101 11001
10100 11011 11100 11001 01000 10001 10011 00000
2 10
0000111001 0110110011
0111110010 0001111000
2 3
010 110
101 001
10 10
1010011110 0001100011 0100100010 0000000101 1010011000 0111111001 1010110110 0101101101 1101011011 1111000100
1000100110 0111010101 1101111100 1111100011 0010111101 0101000001 0110011010 1000100000 0011011011 1000001110
2 6
111001 111111
111001 111111
4 7
1100100 1001011 1101100 1000011
0011010 0110101 0111101 0010010
6 9
010101110 111000011 110110100 100100010 000110110 110001001
111111000 101101101 010110001 101010001 100110000 010001011
7 10
0101011001 0011111111 1001001010 0001010001 0100111011 1010000111 0000110001
1111000000 1101101110 0101110101 0110111000 1000000100 1001100110 1100001110
9 8
11011001 01110010 10000000 10100010 11111011 10010110 10101010 01001110 11000000
11101001 10110000 10010010 01000010 11001011 10100110 11110000 01111110 10011010
6 6
011100 000011 100011 010000 110111 110010
111111 011000 101011 010100 001011 111010
10 9
101000110 101010110 100101001 001000110 000111001 001010110 011000110 000101001 010101001 100111001
010111001 001010110 011000110 010101001 011010110 111010110 111000110 000111001 110111001 110101001
10 10
1110001100 1110100011 0000110010 1001101010 0001011101 0111011100 0001000101 1101110011 0110001011 1010110100
0110000000 1001010001 0010111000 0001100110 0110101111 0101111111 1110000111 1000111110 1111010000 1001001001
10 8
10101010 01111000 01110110 11111100 00110110 10110110 01110010 00111001 00001000 00010101
01001001 10101110 10011000 11001111 00000100 00001101 11010110 00101000 10110100 01010000
8 7
0011111 0111010 0111110 1101110 1101010 1001111 0010110 1000110
0111110 1001111 0110111 1100111 1101110 0011111 1001011 0011011
4 10
0110011100 0001000110 0000010100 0010111111
0000110010 0110111010 0001100000 0010011001
9 8
01101010 00011100 11101000 00100100 11100111 10001011 10001010 10011010 11011010
00000111 10011111 11110001 00011011 01101100 10110011 11100111 01011100 11101111
1 6
000111
000101
9 10
1010001011 1011111110 0110001111 1111010010 1101100001 0111010100 1111111001 1001101101 1001110001
0111100111 0010100001 1001110101 0111101011 1100010001 1111010000 1111010101 0101100110 0000011000
4 7
1010100 0101011 1011101 0100010
1011101 1010100 0100010 0101011
