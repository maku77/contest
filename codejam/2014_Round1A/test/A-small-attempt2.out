Case #1: 1
Case #2: 8
Case #3: 2
Case #4: 2
Case #5: 0
Case #6: 1
Case #7: 4
Case #8: 0
Case #9: NOT POSSIBLE
Case #10: 6
Case #11: 4
Case #12: 2
Case #13: 4
Case #14: 6
Case #15: NOT POSSIBLE
Case #16: NOT POSSIBLE
Case #17: 1
Case #18: 7
Case #19: 5
Case #20: 1
Case #21: NOT POSSIBLE
Case #22: NOT POSSIBLE
Case #23: NOT POSSIBLE
Case #24: 0
Case #25: NOT POSSIBLE
Case #26: 1
Case #27: NOT POSSIBLE
Case #28: 3
Case #29: NOT POSSIBLE
Case #30: 5
Case #31: NOT POSSIBLE
Case #32: 7
Case #33: 6
Case #34: 3
Case #35: NOT POSSIBLE
Case #36: 6
Case #37: 3
Case #38: 3
Case #39: 1
Case #40: 2
Case #41: 4
Case #42: NOT POSSIBLE
Case #43: 1
Case #44: NOT POSSIBLE
Case #45: 2
Case #46: 6
Case #47: 4
Case #48: NOT POSSIBLE
Case #49: 3
Case #50: 1
Case #51: 1
Case #52: 5
Case #53: NOT POSSIBLE
Case #54: 3
Case #55: 5
Case #56: 6
Case #57: 3
Case #58: 2
Case #59: 2
Case #60: NOT POSSIBLE
Case #61: 4
Case #62: NOT POSSIBLE
Case #63: 5
Case #64: 5
Case #65: 1
Case #66: 3
Case #67: NOT POSSIBLE
Case #68: 1
Case #69: 5
Case #70: 1
Case #71: 3
Case #72: NOT POSSIBLE
Case #73: 4
Case #74: 4
Case #75: NOT POSSIBLE
Case #76: 2
Case #77: 8
Case #78: 3
Case #79: NOT POSSIBLE
Case #80: 4
Case #81: 5
Case #82: 2
Case #83: 2
Case #84: 2
Case #85: 4
Case #86: 0
Case #87: 3
Case #88: NOT POSSIBLE
Case #89: 8
Case #90: 2
Case #91: 1
Case #92: 2
Case #93: 3
Case #94: NOT POSSIBLE
Case #95: 2
Case #96: 3
Case #97: NOT POSSIBLE
Case #98: 1
Case #99: NOT POSSIBLE
Case #100: 0
